### ISSUE TITLE
Import from GitHub

### ISSUE THREAD

**Issue by roblevintennis** [stackblitz/core#1821]
TL;DR very opinionated issue feel free to close ;-)

But, `The url to this repository is invalid.` seems to be cached for a few minutes maybe? My feedback would be to bust this cache within 1 minute.

I think I may have hit this just seconds before I'd pushed up the corresponding repo:
https://stackblitz.com/github/AgnosticUI/agnosticui/tree/master/playgrounds/Close?file=/README.md

But then it seemed to have cached the bad repo (I was able to hit the same playground from codesandbox):

![image](https://user-images.githubusercontent.com/142403/161724988-5ae3e449-67ef-4f61-b153-3c57ed2cc2b4.png)

It started working after a few minutes but pretty sure it wasn't 60 seconds which somehow feels like what the cache invalidation should be.

**Comment by HeyGarrison** [comment]
Over the past few years we've made a lot of improvements to our platform. Which has been great! To that end - it looks like this is working now!
